stable/postgresql/templates/_helpers.tpl

```diff
@@ -74,6 +74,133 @@ Also, we can't use a single if because lazy evaluation is not an option
 {{- end -}}
 {{- end -}}
 
+{{/*
+Return PostgreSQL password
+*/}}
+{{- define "postgresql.password" -}}
+{{- if .Values.global }}
```

Comment:
you might be able to reduce the amount of nested if/else cases you need if you set a default for global inside values.yaml that can help ensure that `global.postgresql` is defined:

```
global:
 postgresql: {}
```

Reply:
Done. Thanks!